### PR TITLE
Show Tour Index For User

### PR DESCRIPTION
**Pull request by minhtc-0015**
ready

### CODE REVIEW

**Comment by minhtc-0015** [comment]
ready

**Comment by minhtc-0015** [comment]
ready

File: src/main/java/app/controller/IndexController.java [hunk 1]
```diff
@@ -13,8 +13,8 @@
 import app.model.Tours;
 import app.service.ToursService;
 
-@Controller("CatalogTextureController")
-public class indexController {
+@Controller("IndexForUser")
```
Comment (VuThiTranVan) [inline src/main/java/app/controller/IndexController.java]: `IndexForUser` thành `client/users` hì

**Comment by minhtc-0015** [comment]
ready

**Pull request merged**